packages/Graph/Microsoft.Bot.Components.Graph.csproj

```diff
@@ -17,8 +17,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Graph.Beta" Version="0.38.0-preview" />
     <PackageReference Include="Microsoft.Bot.Builder.Integration.Runtime" Version="4.12.0-rc1.preview" />
-    <PackageReference Include="Microsoft.Graph" Version="3.15.0" />
+    <PackageReference Include="TimeZoneConverter" Version="3.4.0" />
```

Comment:
This needs to come out of all the packages. It should be replaced by the logic in Main